baybe/surrogates/base.py

```diff
@@ -2,29 +2,29 @@
 
 from __future__ import annotations
 
-import gc
-import sys
 from abc import ABC, abstractmethod
 from typing import TYPE_CHECKING, ClassVar
 
 from attrs import define
+from cattrs import override
+from cattrs.dispatch import (
+    StructuredValue,
+    StructureHook,
+    TargetType,
+    UnstructuredValue,
+    UnstructureHook,
+)
 
 from baybe.searchspace import SearchSpace
 from baybe.serialization import SerialMixin, converter, unstructure_base
+from baybe.serialization.core import get_base_structure_hook
```

Comment:
any reason this is not imported from `baybe.serialization` like the others above?

Reply:
No, by mistake. This is one of the things I can also cleanup as part of the import PR, but changed it here nevertheless :+1: